media-plugins/vdr-dvbapi/vdr-dvbapi-2.2.4.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: allows connect VDR to OScam"
+HOMEPAGE="https://github.com/manio/vdr-plugin-dvbapi"
+SRC_URI="https://github.com/manio/vdr-plugin-dvbapi/archive/v${PV}.tar.gz -> vdr-dvbapi-${PV}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+IUSE="cpu_flags_x86_3dnow cpu_flags_x86_mmx cpu_flags_x86_sse cpu_flags_x86_sse2 dvbcsa"
+
+DEPEND=">=media-video/vdr-2.1.4
+	dvbcsa? ( media-libs/libdvbcsa )"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/vdr-plugin-dvbapi-${PV}"
+
+DOCS="HISTORY INSTALL README FFdecsa/docs"
+
+src_prepare() {
+	vdr-plugin-2_src_prepare
+
+	if use dvbcsa; then
+		sed -e "s:PLUGIN = dvbapi:PLUGIN = dvbapi\nLIBDVBCSA = 1:" -i Makefile || die "modifying Makefile"
```

Comment:
Instead of repeating the same thing, you can use `a` command to add an extra line.